.github/workflows/pages.yml

```
@@ -30,7 +30,7 @@ jobs:
       name: github-pages
       url: ${{ steps.deploy.outputs.page_url }}
     steps:
-      - uses: actions/deploy-pages@v1
+      - uses: actions/deploy-pages@v2
```

actionlint (1.7.7)

the runner of "actions/deploy-pages@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)